unite@hardpixel.eu/schemas/org.gnome.shell.extensions.unite.gschema.xml

```diff
@@ -14,8 +14,16 @@
     <value value="11" nick="vertex" />
     <value value="12" nick="pop-dark" />
     <value value="13" nick="pop-light" />
-    <value value="14" nick="vimix" />
-    <value value="15" nick="yaru" />
+<<<<<<< HEAD
```

Comment:
Remove conflict markers and use ids 16 and 17 for the new themes.

Reply:
Done